rails-new.md

```diff
@@ -15,9 +15,9 @@ Instead of having to manage the versions of multiple different dependancies for
 
 In your new directory make a `docker-compose.yml` file and paste in:
 
-```ruby
-# docker-compose.yml
+docker-compose.yml
 --------------------
+```ruby
 
 version: "3"
 services:
```

Comment:
So we are using the format of 
```
# docker-compose.yml
--------------------

Content here below
```

to show that you are in a file titled docker-compose.yml, and below this dotted line you will add **x** content. Maybe we can add a small blurb in the introduction explaining this, but you will see this same method time and time again throughout the book